Source/IncrementableLabel.swift

```diff
@@ -85,6 +88,13 @@ extension IncrementableLabel {
         self.completion = completion
         startIncrementation(fromValue: fromValue, toValue: toValue, duration: duration)
     }
+    
+    public func increment(fromValue: Double, toValue: Double, startColor: UIColor, endColor: UIColor, duration: Double = 0.3, completion: Completion? = nil) {
```

Comment:
A few things here, I think we need to have that new option for all the `increment`
 methods: `increment `, `incrementFromCurrentValue`, `incrementFromZero`

At the same time, I would like to avoid to increase the number of methods to add this option, so what about adding thesee new variables to the existing one as optional, for example:

```swift
    public func increment(fromValue: Double, toValue: Double, startColor: UIColor? = nil, endColor: UIColor? = nil, duration: Double = 0.3, completion: Completion? = nil) {
```